projects/plugins/boost/app/assets/src/js/pages/settings/elements/CriticalCssShowStopperError.svelte

```diff
@@ -8,15 +8,18 @@
 	import { __ } from '@wordpress/i18n';
 	import ErrorNotice from '../../../elements/ErrorNotice.svelte';
 	import FoldingElement from '../../../elements/FoldingElement.svelte';
-	import { criticalCssState, regenerateCriticalCss } from '../../../stores/critical-css-state';
-	import { primaryErrorSet } from '../../../stores/critical-css-state-errors';
+	import { regenerateCriticalCss } from '../../../stores/critical-css-state';
```

Comment:
Do we want to pass `regenerateCriticalCSS` as a prop as well since this also interacts with the state?

Reply:
I've moved the function to the Modules component in b6ddc19.

Looking at the whole chain - there's something about passing down through 2 layers that I don't like.

However, I'm not sure how to improve it \: Any suggestions? Maybe event dispatching? But that would still have to bubble up...